README.md

```diff
@@ -4,6 +4,54 @@
 
 This project contains very opinionated packages for common operations.
 
+## Request
+
+### Usage
+
+It will be used the following service as example:
+
+```golang
+type Service interface {
+    Do(context.Context, Request) (Response, error)
+}
+
+type Response struct {
+    //(...)
+    RequestID request.ID
+}
+```
```

Comment:
Why recommend the Request ID be passed by struct if you added the GetRequestIDFromContext? This could be the standard way to manipulate Request IDs.

Reply:
When you are encoding the response, you don't have the service's context, only the service response. Could be used in "BaseEndpointResponse", but this project don't have a standard API base response (yet)